packages/terra-application-layout/docs/usage.md

```diff
@@ -0,0 +1,278 @@
+# Terra Application Layout
+
+## Prerequisites
+
+- Like all Terra components, the ApplicationLayout requires the presence of a `Base` component (provided by `terra-base`) in its parent hierarchy.
+- Additionally, the ApplicationLayout requires the presence of any `Router` component (provided by `react-router-dom`) in its parent hierarchy.
+
+## Props
+
+### `nameConfig`
+
+The `nameConfig` prop allows consuming applications to add their own branding to the ApplicationLayout. The ApplicationLayout will render this content in various ways based on the current breakpoint.
+
+|Key Name|Type|Is Required|Description|
+|---|---|---|---|
+|`accessory`|Element|optional|A React element that will be presented beside the title. This is typically some sort of icon or visual branding.
+|`title`|String|optional|The title of the application. This text will be styled by the ApplicationLayout.|
+
+```jsx
+const myNameConfig = {
+  accessory: <MyApplicationLogo />,
+  title: 'My Application',
+}
+```
+
+### `utilityConfig`
+
+The `utilityConfig` prop allows consuming applications to present an application-level menu directly from the ApplicationLayout. The ApplicationLayout will render this content in various ways based on the current breakpoint.
+
+The API for the `utilityConfig` matches that of the `ApplicationHeaderUtility` and `ApplicationMenuUtility` components from `terra-application-utility`. Please see their respective documentation for more information regarding their full capabilities.
+
+|Key Name|Type|Is Required|Description|
+|---|---|---|---|
+|`title`|String|optional|A string rendered within the utility menu's presentation target.|
+|`accessory`|Element|optional|A React element rendered next to the title.|
+|`menuItems`|Array|required|An array of objects specifying the utility menu options to present.|
+|`selectedKey`|String|required|The string key of the initial menu item to present.|
+|`onChange`|Function|required|A function that will be called upon selection of a terminal utility item.|
+
+```jsx
+import Avatar from 'terra-avatar';
+import { UtilityUtils } from 'terra-application-utility';
+
+const myUtilityConfig = {
+  title: 'Doe, John',
+  accessory: <Avatar variant="user" ariaLabel="Doe, John" />,
+  menuItems: [{
+    key: 'menu',
+    title: 'Menu',
+    childKeys: ['item-1', 'item-2', 'item-3', 'item-4'],
+  }, {
+    key: 'item-1',
+    title: 'Item 1',
+  }, {
+    key: 'item-2',
+    title: 'Toggle Item - Checked',
+    isSelectable: true,
+    isSelected: true,
+  },  {
+    key: 'item-3',
+    title: 'Toggle Item - Not Checked',
+    isSelectable: true,
+    isSelected: false,
+  }, {
+    key: 'item-4',
+    contentLocation: UtilityUtils.LOCATIONS.FOOTER,
+    title: 'Footer Item',
+  }]),
```

Comment:
There is an unneeded `)` here that can be removed

Reply:
Nice spot